### PR TITLE
Added Pull Request template

### PR DESCRIPTION
**Pull request by VaishnaviKillekar**
Added a PR template so all subsequent PRs can use the provided template to include relevant details.

### CODE REVIEW

**Review by omkarajagunde — APPROVED** [review]
LGTM...

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Enhancement (improved an existing feature)
+- [ ] This change requires a documentation update
```
Comment (Ashwand) [inline .github/pull_request_template.md]: If we have to  delete options that are not relevant, I think it's better not to have check box

Comment (VaishnaviKillekar) [reply .github/pull_request_template.md]: I can update the text so we can simply check the appropriate option

**Review by Ashwand — APPROVED** [review]

**Comment by ashrithshetty** [comment]
Looks goo

**Pull request merged**